src/main/java/pl/wojciechkostecki/revel/exception/MenuIsAlreadyAssignedException.java

```diff
@@ -0,0 +1,9 @@
+package pl.wojciechkostecki.revel.exception;
+
+import java.text.MessageFormat;
+
+public class MenuIsAlreadyAssignedException extends RuntimeException{
```

Comment:
Instead of using this exception you could've just used `BadRquestException`. When you want to use some custom exceptions I would recommend searching for recommended spring/java exceptions for your case.